src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -10,26 +10,52 @@
 
 #define FLAGS (ACTOR_FLAG_0 | ACTOR_FLAG_3)
 
+#define TALON_STATE_FLAG_TRACKING_PLAYER (1 << 0)
+#define TALON_STATE_FLAG_GIVING_MILK_REFILL (1 << 1)
+#define TALON_STATE_FLAG_SUPPRESS_BLINK (1 << 2)
+#define TALON_STATE_FLAG_3 (1 << 3)
+#define TALON_STATE_FLAG_ANIMATION_FINISHED (1 << 4)
+#define TALON_STATE_FLAG_CUCCO_GAME_START_EVENT_TRIGGERED (1 << 5)
+#define TALON_STATE_FLAG_UNUSED (1 << 6)
+#define TALON_STATE_FLAG_FLOOR_CAMERA_ACTIVE (1 << 7)
+#define TALON_STATE_FLAG_RAISING_HANDS (1 << 8)
+#define TALON_STATE_FLAG_RESTORE_BGM_ON_DESTROY (1 << 9)
+
+#define TALON_FACE_REACTION_SET 24
+
+#define EVENTINF_CUCCO_GAME_FINISHED EVENTINF_HORSES_0A
+#define EVENTINF_CUCCO_GAME_WON EVENTINF_HORSES_08
+
+typedef enum { TALON_EYE_INDEX_OPEN, TALON_EYE_INDEX_HALF, TALON_EYE_INDEX_CLOSED } TalonEyeIndex;
```

Comment:
We prevent the formatter from putting such enums on single lines by having comments like `/* 0 */` before each (one example among many: `EnDaikuTalkState`), and in general we do that on all enums

Reply:
Ok, I will add those!